inst/Description.qml

```diff
@@ -14,9 +14,31 @@ Description
 	license:		"GPL (>= 2)"
 		
 
+  GroupTitle
```

Comment:
tabs in QML, not spaces

Reply:
Okay I tried to change this using VS Code, hope it worked but let me know if not!